esphome/components/i2s_audio/media_player/i2s_audio_media_player.h

```diff
@@ -71,6 +73,8 @@ class I2SAudioMediaPlayer : public Component, public media_player::MediaPlayer,
 #endif
   uint8_t external_dac_channels_;
 
+  bool i2s_comm_fmt_;
```

Comment:
Maybe rename this and the setter to be more descriptive
```suggestion
  bool i2s_comm_fmt_lsb_;
```

Reply:
Done!